interfaces/apparmor/template.go

```diff
@@ -319,6 +319,10 @@ var defaultTemplate = []byte(`
   # access in /dev/shm for shm_open() and files in subdirectories for open()
   /{dev,run}/shm/snap.@{SNAP_NAME}.** mrwlkix,
 
+  # Snap-specific XDG_RUNTIME_DIR that is based on the UID of the user
```

Comment:
@zyga - I don't understand this comment: "Can we use + rather than *, just to ensure that component exists please." in the context of where you put the comment. Can you elaborate? (the apparmor rule is correct)